### PR TITLE
add SPDX-License-Identifier and use shorter license header

### PR DESCRIPTION
**Pull request by ghost**
to satisify with the requirements of reuse tool

### CODE REVIEW

**Comment by ghost** [comment]
retest this please

File: include/common/monitor_config.h [hunk 1]
```diff
@@ -3,17 +3,6 @@
 *
 * SPDX-License-Identifier: Apache-2.0
 *
```
Comment (thakurajayL) [inline include/common/monitor_config.h:5]: We dont want to put ONF name ?

Comment (ghost) [reply include/common/monitor_config.h:5]: good point, I will add

Comment (ghost) [reply include/common/monitor_config.h:5]: done

**Comment by ghost** [comment]
retest this please

**Comment by ghost** [comment]
retest this please

**Review by thakurajayL — APPROVED** [review]

**Review by sindack — APPROVED** [review]

**Pull request merged**